capstone-server/src/main/java/com/hcl/capstoneserver/invoice/entities/Invoice.java

```diff
@@ -0,0 +1,126 @@
+package com.hcl.capstoneserver.invoice.entities;
+
+import com.hcl.capstoneserver.invoice.CurrencyType;
+import com.hcl.capstoneserver.user.entities.Client;
+import com.hcl.capstoneserver.user.entities.Supplier;
+
+import javax.persistence.*;
+import java.util.Arrays;
+
+@Entity
+public class Invoice {
+    @Id
+    @GeneratedValue
+    private Integer invoiceId;
+    @ManyToOne
+    @JoinColumn(name = "CLIENT_ID", nullable = false)
+    private Client clientId;
+    @ManyToOne
+    @JoinColumn(name = "SUPPLIER_ID", nullable = false)
+    private Supplier supplierId;
+    @Column(unique = true)
```

Comment:
remove this. it is possible to for two clients to have the same invoice number. since we have invoiceId as a unique field. this is not needed